server/users/users.controller.ts

```diff
@@ -19,8 +24,14 @@ class UsersController {
 		const userId: number = Number(req.params.id);
 
 		try {
-			const findOneUserData = await this.userService.getById(userId);
-			res.status(200).json({ data: findOneUserData, message: 'findOne' });
+			const userData = await this.userService.getById(userId);
+
+			res.send(UserPage({
+				email: userData.email,
```

Comment:
Why not just return the whole `userData` object if all the data is there? As everything is rendered server side, we don't have to worry about leaking too much info to the frontend.

Reply:
Changed in https://github.com/mbrizic/ratethegame/pull/28/commits/e182b93143ecc6d9284b7af89712444994a49dbe